modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/OverriddenSubstitutionGroupTestCase.java

```diff
@@ -24,7 +24,7 @@ public void testSchemaIsCorrectlyGenerated() throws Exception {
     try {
       flowRunner("doNothing").run();
```

Comment:
why do the try/catch at all?

Reply:
So that the test is reported as failed assertion (yellow) instead of error (red), it was written before that way.